cmd/util/cmd/check-storage/cmd.go

```diff
@@ -30,6 +30,18 @@ var (
 	flagNWorker         int
 )
 
+var storageMapDomains = []string{
```

Comment:
We have this list now in several places. We should maybe export this from Cadence and use it everywhere in flow-go

Reply:
> We have this list now in several places. We should maybe export this from Cadence and use it everywhere in flow-go

That is a great idea! :+1:   If you open PR in Cadence to export storage map domains, then I can open a flow-go PR to use that.

Is the window still open to update Cadence in flow-go v0.37?  We probably want to port this PR to support `cap_tag` in flow-go v0.37 before spork.